alot/buffers.py

```diff
@@ -85,7 +85,7 @@ def rebuild(self):
             self.isinitialized = True
 
         lines = list()
-        displayedbuffers = filter(self.filtfun, self.ui.buffers)
```

Comment:
I can go either way on the ``filter`` and ``map`` calls that don't have a lambda (the comprehensions are quite a bit faster if we can drop a lambda), I personally like the consistancy of doing it one way, but they are more verbose. There is one thing to think about as we look to python 3, ``map`` and ``filter`` return iterators in python 3 (they're basically ``itertools.imap`` and ``itertools.ifilter``), while comprhensions are the same between the two (other than that list comrehensions leak the last value calculated in python 2).